### ISSUE TITLE
Exporting image crashes dt on Win10

### ISSUE THREAD

**Issue by juhavlintula** [darktable-org/darktable#7222]
**Describe the bug**
Exporting an image as a JPG crashes dt on Windows 10

**To Reproduce**
1. Open a folder
2. Select an image
3. Export an image
I have had different experiences, sometimes I have managed to export a single image, but when I change the export settings (image size, JPG quality)
4. Crash (dump attached)

**Expected behavior**
No crash

**Screenshots**
–

**Platform (please complete the following information):**
 - Darktable Version: 3.3.0-1993
 - OS: Windows10/64

**Additional context**
I did remove the build directory and regenerated everything and that didn't remove the issue.
[darktable_bt_1O3WU0.txt](https://github.com/darktable-org/darktable/files/5649195/darktable_bt_1O3WU0.txt)

**Comment by TurboGit** [comment]
> Exporting an image as a JPG crashes dt on Windows 10

Of course not otherwise dt won't be usable on Windows.

We will need far more information. The modules activated, the computer, RAW, graphic card if any... First try to do on Windows if you don't have a graphic card is to disable OpenCL as implementation on Intel is buggy.

**Comment by Nilvus** [comment]
Seems quite similar to  #7218 and so probably a broken config.

**Comment by juhavlintula** [comment]
I run this with full debug, and the last module executed is filmicrgb. 
[darktable-log.txt](https://github.com/darktable-org/darktable/files/5649308/darktable-log.txt)
```
50,564262 [opencl_filmicrgb] couldn't enqueue kernel! -4
50,564262 [opencl_pixelpipe] could not run module 'filmicrgb' on gpu. falling back to cpu path
```
I have both
- GeForce 940MX and
- Intel(R) HD Graphics 630
darktable-cltest reports Intel as blacklisted.

Running with option `disable-opencl` crashes darktable immediately, when I open a folder with scene-referred processed images.

**Comment by juhavlintula** [comment]
I tested this with version 2031 on my virtual-Ubuntu, and no problems. As the GPU is not exposed to VMs OpenCL and OpenCL choices in the preferences were disabled.

I retested in Win10 and opening a folder and editing an image works, when I have OpenCL enabled, but exporting an image crashes dt. If I disable OpenCL in the preferences (or on command line), opening a folder crashes dt. The Windows crash log looks similar in both cases pointing to filmicrgb.dll.

**Comment by juhavlintula** [comment]
I have done some testes on the previous versions and the commit 251e0a3 works, but when compiling with commit 1522dc5 I get crash.

**Comment by phrrk** [comment]
With Win10 and version 2059 I can succesfully export to jpeg, with or without OpenCL, a randomly chosen image that includes filmicrgb amongst the processing modules.   Also I do not see anything like:
```
50,564262 [opencl_filmicrgb] couldn't enqueue kernel! -4
50,564262 [opencl_pixelpipe] could not run module 'filmicrgb' on gpu. falling back to cpu path
```
in the logfile.  See:

```
15.513632 [dev_pixelpipe] took 0.085 secs (0.000 CPU) processed `filmic rgb' on GPU, blended on GPU [export]
[dev_pixelpipe] module `filmic rgb' min: (0.000151; 0.000000; 0.000000) max: (0.966386; 0.970675; 0.988551) [export]
```
This is with a GTX1060

**Comment by juhavlintula** [comment]
I retested with 2067 and I still get the same errors.
1. When I try to export a group of images, dt crashes on the first one
2. If I disable OpenCL in the preferences and open a folder of 13 images, I get 10 thumbnails and three error windows telling dt has crashed.

**Comment by juhavlintula** [comment]
Both my cases work now on 2108. I will close the issue and reopen it if I encounter it again.

**Comment by ralfbrown** [comment]
Probably the same cause as #7273, which was fixed by #7292.
